src/Common/src/Interop/Windows/NtDll/Interop.NtCreateFile.cs

```diff
@@ -50,24 +53,28 @@ internal partial class NtDll
                     rootDirectory);
 
                 int status = NtCreateFile(
-                    out IntPtr handle,
-                    desiredAccess,
-                    ref attributes,
+                    FileHandle: out IntPtr handle,
```

Comment:
nit: These aren't necessary, in fact you're missing one for statusBlock. The only reason I added them later was to be explicit about what null was (AllocationSize:).  In earlier versions of C# you had to name all of the arguments after named one. This really should only have just that one explicit- it helps when reading the code when explicitly calling out what arguments are when passing things like null, true, false, and other unnamed constant values. (So you don't have to look at the called method to understand what the value is.)

Reply:
Changed code to only one explicit `AllocationSize` as the other arguments are self descriptive.